contrib/storage-http/src/test/java/org/apache/drill/exec/store/http/TestHttpPlugin.java

```diff
@@ -140,12 +141,17 @@ private static void makeMockConfig() {
     HttpApiConfig mockXmlConfig = new HttpApiConfig("http://localhost:8091/xml", "GET", headers,
       "basic", "user", "pass", null, null, "results", null, "xml", 2,false);
 
+    HttpApiConfig mockGithubWithParam = new HttpApiConfig("http://localhost:8091/orgs/{org}/repos", "GET", headers,
```

Comment:
Please add tests for the various cases above, along with:

* Duplicate names
* Mixed-case names (param="org", column="ORG", etc.)
* Missing column name
* Column of the wrong type
* Parameter is in the query clause (something that should be tested anyway, url: "...?foo=bar"

Reply:
Added additional unit tests for these cases.